src/main/java/book/store/service/shoppingcart/ShoppingCartService.java

```diff
@@ -0,0 +1,16 @@
+package book.store.service.shoppingcart;
+
+import book.store.dto.cartitem.CartItemDto;
+import book.store.dto.cartitem.CreateCartItemRequestDto;
+import book.store.dto.cartitem.UpdateCartItemRequestDto;
+import book.store.dto.shoppingcart.ShoppingCartDto;
+
+public interface ShoppingCartService {
+    ShoppingCartDto getCartWithCartItems(Long userId);
+
+    ShoppingCartDto addCartItemToCart(Long userId, CreateCartItemRequestDto requestDto);
+
+    CartItemDto updateCartItemQuantityInCart(UpdateCartItemRequestDto requestDto, Long cartItemId);
```

Comment:
`updateCartItemQuantity` is enough, also please make sure that the args order is consistent, for example: first pass the path param, then the request body object